README.md

```diff
@@ -67,4 +67,4 @@ After:
 * Now if you navigate to `http://app-renderer-devxenial.mntd.net/render/footer` you will see your new repos application.
     Note: To have your JS working in `http://app-renderer-devxenial.mntd.net/render/footer` you will need to run this repos server with `yarn dev` 
 
-* To view any new code changes on `http://app-renderer-devxenial.mntd.net/render/footer` you will need to rebuild the app-renderer-server bundle and manually refresh the app-renderer page. 
+* To view any new code changes from your new repo on `http://app-renderer-devxenial.mntd.net/render/footer` or `https://web-devxenial.mntd.net/...`, all you need to do is have your repos `yarn dev` running and then manually refresh the app-r url.
```

Comment:
You shouldn't need to manually refresh App-Renderer if you're running the dev server because we've got FastRefresh setup. It hot-reload itself. Please test and verify. We might have to include the correct a host address to ensure it's not pointing at `window.location`.

Reply:
the app-r browser does not refresh itself. the reason why i used `/render/footer` is because the example in the readme is specific to footer

Reply:
I think it's confusing to use `/footer` when you'll have to configure App-Renderer for your app anyway.

Also, App-Renderer won't refresh, correct. But FastRefresh should still work regardless.